turbopack/crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -2714,9 +2712,14 @@ pub async fn handle_resolve_error(
     origin_path: Vc<FileSystemPath>,
     request: Vc<Request>,
     resolve_options: Vc<ResolveOptions>,
-    severity: Vc<IssueSeverity>,
+    is_optional: bool,
     source: Option<Vc<IssueSource>>,
 ) -> Result<Vc<ModuleResolveResult>> {
+    let severity = if is_optional || resolve_options.await?.loose_errors {
+        IssueSeverity::Warning.cell()
+    } else {
+        IssueSeverity::Error.cell()
+    };
```

Comment:
Most resolve calls won't lead to errors, so better move that down to only execute when there is an error. Maybe move it into `emit_unresolveable_issue` and `emit_resolve_error_issue`